From 33d1e856d50618fab5a1906d0596245bd149db8d Mon Sep 17 00:00:00 2001
From: Zheng Yejian <zhengyejian1@huawei.com>
Date: Mon, 22 Aug 2022 20:23:55 +0000
Subject: [PATCH 027/261] [Huawei] livepatch: Fix warning
 C_RULE_ID_PRINT_NUMBER_DONT_USE_BRACKET

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 187321

--------------------------------

CodeCheck reports that:
  Do not use parentheses when printing numbers.

Signed-off-by: Zheng Yejian <zhengyejian1@huawei.com>
---
 arch/arm64/kernel/livepatch.c      | 4 ++--
 arch/powerpc/kernel/livepatch_32.c | 4 ++--
 arch/powerpc/kernel/livepatch_64.c | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/kernel/livepatch.c b/arch/arm64/kernel/livepatch.c
index 42b4caa225fb..5ebd4d5279df 100644
--- a/arch/arm64/kernel/livepatch.c
+++ b/arch/arm64/kernel/livepatch.c
@@ -417,7 +417,7 @@ static int do_patch(unsigned long pc, unsigned long new_addr)
 		for (i = 0; i < LJMP_INSN_SIZE; i++) {
 			ret = aarch64_insn_patch_text_nosync(((u32 *)pc) + i, insns[i]);
 			if (ret) {
-				pr_err("patch instruction(%d) large range failed, ret=%d\n",
+				pr_err("patch instruction %d large range failed, ret=%d\n",
 				       i, ret);
 				return -EPERM;
 			}
@@ -463,7 +463,7 @@ void arch_klp_unpatch_func(struct klp_func *func)
 			ret = aarch64_insn_patch_text_nosync(((u32 *)pc) + i,
 							     func_node->arch_data.old_insns[i]);
 			if (ret) {
-				pr_err("restore instruction(%d) failed, ret=%d\n", i, ret);
+				pr_err("restore instruction %d failed, ret=%d\n", i, ret);
 				return;
 			}
 		}
diff --git a/arch/powerpc/kernel/livepatch_32.c b/arch/powerpc/kernel/livepatch_32.c
index 3b9e720983ca..7b4ed23bf2ca 100644
--- a/arch/powerpc/kernel/livepatch_32.c
+++ b/arch/powerpc/kernel/livepatch_32.c
@@ -431,7 +431,7 @@ static int do_patch(unsigned long pc, unsigned long new_addr)
 			ret = patch_instruction((struct ppc_inst *)(((u32 *)pc) + i),
 						ppc_inst(insns[i]));
 			if (ret) {
-				pr_err("patch instruction(%d) large range failed, ret=%d\n",
+				pr_err("patch instruction %d large range failed, ret=%d\n",
 				       i, ret);
 				return -EPERM;
 			}
@@ -469,7 +469,7 @@ void arch_klp_unpatch_func(struct klp_func *func)
 			ret = patch_instruction((struct ppc_inst *)(((u32 *)pc) + i),
 						ppc_inst(func_node->arch_data.old_insns[i]));
 			if (ret) {
-				pr_err("restore instruction(%d) failed, ret=%d\n", i, ret);
+				pr_err("restore instruction %d failed, ret=%d\n", i, ret);
 				return;
 			}
 		}
diff --git a/arch/powerpc/kernel/livepatch_64.c b/arch/powerpc/kernel/livepatch_64.c
index 8c7b44acf74e..416f9f03d747 100644
--- a/arch/powerpc/kernel/livepatch_64.c
+++ b/arch/powerpc/kernel/livepatch_64.c
@@ -467,7 +467,7 @@ void arch_klp_unpatch_func(struct klp_func *func)
 			ret = patch_instruction((struct ppc_inst *)((u32 *)pc + i),
 						ppc_inst(func_node->arch_data.old_insns[i]));
 			if (ret) {
-				pr_err("restore instruction(%d) failed, ret=%d\n", i, ret);
+				pr_err("restore instruction %d failed, ret=%d\n", i, ret);
 				break;
 			}
 		}
-- 
2.17.1

